rchorngren/whs
# Layout ground work

# Added basic layout for the app
- Header
- Side menu
- Main content area
- Footer

### How to test:
- Clicking each button should generate a "click effect" as well as displaying the corresponding text in the main area of the app (between the header and footer)
- Clicking the menu button should display the side menu overlapping the main area
- Clicking any button (including the menu button itself) while the side menu is open should close the side menu and reveal the same component text that was active prior to opening the menu

### ToDo:
- Animation when opening/closing side menu
- Styling in general (should be responsive already though)
- Connect various future redux based values, such as if the user is logged in or not, number of items in basket etc.

### Known issue:
- The following console message is displayed (doesn't seem to break anything though but should be fixed):
*src\Components\ActiveView.jsx
 Line 62:8:  React Hook useEffect has a missing dependency: 'content'. Either include it or remove the dependency array  react-hooks/exhaustive-deps*
